### PR TITLE
Story/gh371 notifications dialog

### PR DESCRIPTION
**Pull request by bakerac4**
Fixes front end for #371

### CODE REVIEW

File: pep/app/pods/components/modal-dialogs/whats-new/subscription/template.hbs [hunk 1]
```diff
@@ -1,46 +1,33 @@
...
-                @action={{validator.submit}}
-            />
-        </Modal.footer>
-    </FormValidator>
+    <Modal.body>
+        <p class="text-muted">
+            {{t "relatedInfo.widgets.whatsNew.signUp"}}
+        </p>
+        <Alert
+            @isShown={{bool this.loginError}}
```
Comment (billdami) [inline pep/app/pods/components/modal-dialogs/whats-new/subscription/template.hbs:8]: do we want to show this alert here if there is a login error? looks like maybe unrelated code.

File: pep/app/pods/user/model.ts [hunk 2]
```diff
@@ -24,19 +26,60 @@ interface ActiveSubscription {
...
+    subscriptionEndDate!: Date;
+    @alwaysSend
+    @attr('string')
+    userFullName!: string;
+    @alwaysSend
+    @attr('string')
+    userName!: string;
+    @alwaysSend
+    @attr('string')
+    userType!: UserType;
```
Comment (billdami) [inline pep/app/pods/user/model.ts:82]: this is alot harder to scan down the list to read model attributes now, imo. if we dont want to keep the single line decorator format here, e.g. `@alwaysSend @attr('string') userName!: string;`, i think we should at least have an empty line between each attribute when using newline-separated decorator format.

**Review by billdami — CHANGES_REQUESTED** [review]

File: pep/app/pods/user/model.ts (hunk 2 above)
Comment (bakerac4) [reply pep/app/pods/user/model.ts:82]: This is how `prettier` automatically formats it. I can add the empty line though

Comment (billdami) [reply pep/app/pods/user/model.ts:99]: not very pretty, prettier. 😤

**Review by billdami — APPROVED** [review]
